### DISCUSSION TITLE
Egyszerűsített számla nyomtatás - javaslat

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#9] [Ötletek]
A 2023.04.18-i, egyeztetésen bemutatott „bizonylat típusok” között szerepel a papír alapú, kötelezően nyomtatandó „egyszerűsített számla és annak módosítása, érvénytelenítése” bizonylatok. A jelenlegi szabályzás (48/2013 (XI. 15.) NGM rendelet) szerint nyugtanyomtatón kinyomtatott egyszerűsített számla minimális szélessége 70 mm. Javaslom ennek módosítását 57 mm-re, mint az „egyszerű” pénztárgépekben leggyakrabban előforduló papírméret.
 
Az ÁFA törvény szerinti minimális formai elvárás a számlát illetően, hogy jól olvasható legyen és  ez 57 mm-es papíron is maximálisan teljesül
Amennyiben nem pénztárgépen nyomtatok egyszerűsített számlát, szintén semmilyen elvárás nincs papírszélességet illetően
Kisebb boltok, kitelepülések, hordozható pénztárgéphasználók esetében ezáltal elkerülhető a több eszköz vagy papíralapú egyszerűsített számla használata

Kérem a javaslatom megfontolását.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
